go.mod

```diff
@@ -45,7 +45,7 @@ require (
 	golang.org/x/mod v0.22.0
 	golang.org/x/net v0.31.0
```

Comment:
https://pkg.go.dev/vuln/GO-2024-3333
If you could bump this to v0.33.0 as well here that would be cool.